Jenkinsfile

```diff
@@ -47,12 +81,15 @@ if (launchUnitTests.equals("yes")) {
     stage("Unit tests and Code style") {
         def tasks = [:]
 
-        tasks["php-cs-fixer"] = {runPhpCsFixerTest()}
+        String currentClient = "php-http/guzzle6-adapter"
+        String currentPsrImplem = "guzzlehttp/psr7"
 
-        for (phpVersion in supportedPhpVersions) {
-            def currentVersion = phpVersion
+        tasks["php-cs-fixer"] = {runPhpCsFixerTest("7.1", currentClient, currentPsrImplem)}
```

Comment:
phpcs rules can be different from one version to another, so, personaly, I'd put it for each supported version but it's as you which